test/e2e/operator_test.go

```diff
@@ -35,7 +36,11 @@ import (
 	gatewayv1alpha1 "sigs.k8s.io/gateway-api/apis/v1alpha1"
 )
 
-const timeout = 3 * time.Minute
+const (
+	timeout = 3 * time.Minute
+	// latestRelease is the "from" version used to test operator upgrades.
+	latestRelease = "v1.16.0"
```

Comment:
is this now needed because the upgrade test is also upgrading CRDs, so using "latest" won't work since you need to know the specific version to pull from GH? Just making sure I'm following the change.